sys/page_revisions.js

```diff
@@ -679,10 +681,18 @@ PRS.prototype.getRevision = function(hyper, req) {
             }
         });
     }
+
+    function fetchFromMW() {
```

Comment:
Yep, when we're using it from `getTitleRevison` we don't wanna check because it's already checking afterwards.